pkgs/applications/video/mpv/scripts/autoload.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchurl, lib }:
+
+stdenv.mkDerivation rec {
+  pname = "mpv-autoload";
+  version = "2019-02-15";
+  src = fetchurl {
+    url = "https://raw.githubusercontent.com/mpv-player/mpv/e162bcb5a09b5ad514636684314e8e797ab2a47b/TOOLS/lua/autoload.lua";
+    sha256 = "0fs04dzz7zcprprsmhnwzn6ma6x6v3jkyx8yyf0rdcna51ndrxbk";
+  };
```

Comment:
Now I noticed, you are extracting this from mpv's source. How about using mpv's version and perhaps even `mpv.src` to get the lua script? This should save you the trouble of dealing with updates to this derivation.